images/assets/patches/0011-ocistorage-backend-changes.patch

```diff
@@ -1,32 +1,17 @@
-From 63ffa36a6eaf2c7cae84c91070228b6b1afd9bcf Mon Sep 17 00:00:00 2001
+From 216357d83b3061a2fbbea04a4110199e5a75e98b Mon Sep 17 00:00:00 2001
 From: Dennis Kliban <dkliban@redhat.com>
 Date: Thu, 7 Nov 2024 15:39:12 -0500
 Subject: [PATCH] ocistorage backend changes
 
 ---
- pulpcore/app/models/domain.py      |  2 +-
- pulpcore/app/serializers/domain.py |  9 ++++++++-
+ pulpcore/app/serializers/domain.py | 16 +++++++++++++++-
  pulpcore/app/settings.py           |  1 +
  pulpcore/constants.py              |  1 +
- pulpcore/content/handler.py        |  7 +++++++
- pulpcore/tasking/tasks.py          | 29 +++++++++++++++++++++++++++--
- 6 files changed, 45 insertions(+), 4 deletions(-)
+ pulpcore/content/handler.py        |  2 ++
+ 4 files changed, 19 insertions(+), 1 deletion(-)
 
-diff --git a/pulpcore/app/models/domain.py b/pulpcore/app/models/domain.py
-index 18dd3e892..2fc79e09f 100644
---- a/pulpcore/app/models/domain.py
-+++ b/pulpcore/app/models/domain.py
-@@ -39,7 +39,7 @@ class Domain(BaseModel, AutoAddObjPermsMixin):
-     description = models.TextField(null=True)
-     # Storage class is required, optional settings are validated by serializer
-     storage_class = models.TextField(null=False)
--    storage_settings = EncryptedJSONField(default=dict)
-+    storage_settings = EncryptedJSONField(default=dict, blank=True)
```

Comment:
Does it need to have a `blank=True` here?

Reply:
I removed the blank=True because the storage backend now requires settings so we don't need to support not providing settings.